proposal/more-math-functions.md

```diff
@@ -45,26 +45,27 @@ digits: `3.141592654`.
 clamp($min, $number, $max)
 ```
 
-* Throw an error if the units of `$min`, `$number`, and `$max` are not
-  [compatible][] with each other.
-* Return `$min` if `$min >= $max`
-* Return `$min` if `$number <= $min`.
-* Return `$max` if `$number >= $max`.
+* If the units of `$min`, `$number`, and `$max` are not [compatible][] with each
+  other, throw an error.
+* If `$min >= $max`, return `$min`.
+* If `$number <= $min`, return `$min`.
+* If `$number >= $max`, return `$max`.
 * Return `$number`.
 
 [compatible]: ../spec/built_in_modules/math.md#compatible
 
 ### `hypot()`
 
 ```
-hypot($numbers...)
+hypot($arguments...)
 ```
 
-* Throw an error if any argument is not a [length][].
-* Return `Infinity` if any argument is `Infinity`.
-* Return the square root of the sum of the squares of each argument.
-
-[length]: https://drafts.csswg.org/css-values-4/#lengths
+* If all arguments are not compatible with each other, throw an error.
+* Let the unit of the return value be the same as that of the leftmost argument
+  that has a unit. If all arguments are unitless, let the return value be
+  unitless.
```

Comment:
What do you think about forbidding mixing unitless numbers with numbers that have units? The lack of transitivity there is weird, and while it's probably too late to change it for `+` and `-`, we could definitely change it here.

Reply:
The only reason I considered allowing unitless numbers was to model the behavior after `+`/`-` 🙂 But if the lack of transitivity is undesired, I'm good with forbidding mixing unitless/units.